src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs

```diff
@@ -136,6 +136,18 @@ public void TestAzureDiskEncryptionExtensionSinglePass()
             ComputeTestController.NewInstance.RunPsTest(_logger, "Test-AzureDiskEncryptionExtensionSinglePass");
         }
 
+#if NETSTANDARD
+        [Fact(Skip = "Updated Storage, needs re-recorded")]
+        [Trait(Category.RunType, Category.DesktopOnly)]
+#else
+        [Fact(Skip = "TODO: only works for live mode due to key vault dependency")]
```

Comment:
Why would it only work for live mode?  KeyVault is just as mockable as network is.